documentation/src/main/resources/templates/all-properties.ftl

```diff
@@ -2,7 +2,7 @@
 
 # All Properties
 
-This page lists all [solution_name] properties including advanced and deprecated properties.
+This page lists all [solution_name] properties including advanced and deprecated.
```

Comment:
Kind of ambiguous. Like as if 'advanced' and 'deprecated' are themselves properties.

Reply:
Tweak made to the language there. Thank you.